### ISSUE TITLE
Best way to keep most up-to-date location?

### ISSUE THREAD

**Issue by DeveloperJoshua14** [pnbruckner/ha-life360#37]
System Info:
_Core -> 2024.11.1
Operating System -> 13.2
Supervisor -> 2024.11.2
Frontend -> 20241106.2_

Hello! First of all, I would like to thank you so much for your work on this project, I know we are a small (ish) group that uses this, but I truly appreciate it. 

Currently, I have a automation that runs every 30 seconds to call `life360.update_location`. I know this is awful, but it does not (Or has not for me) updated locations if I do not call this. I am using it for location history, hence why I need the location to update a lot (I would be happy if it only updated every 30 second, but I do not know how to do that.)

Thank you for listening to my "issue", I just wanted to ask. (:

Thanks again

**Comment by pnbruckner** [comment]
You should probably read Life360's FAQs and see if there are any suggestions for getting their app to update location more often in the background. This integration just polls the server for any available updates. It does not normally request updates. So, if you're not seeing updates without using that service, then the issue is with the Life360 app on your devices, not this integration. FWIW, I've never had this problem, and I don't think most users have either. Having said that, I don't think your experience is unheard of, though. Also, in my experience, I use the integration with four devices, two Android and two iPhone. The Android phones definitely update more frequently than the iPhones.

Just an FYI. As best I can tell, whenever you open the Life360 app on one of the phones in the Circle, all phones in the Circle will start updating their location very frequently. But after you close the app, they all (eventually) go back to their normal update behavior, which of course, is designed to use as little battery as possible. That usually means updates can be very far apart (a half hour or more) if the phone is not moving.

Bottom line, as a reminder, the Life360 API we're using is not documented or supported, so it's very difficult to know exactly how to use it.
